components/filter/filter-dimension-set.js

```diff
@@ -66,6 +84,17 @@ class FilterDimensionSet extends LitElement {
 		}));
 	}
 
+	_getSearchType() {
```

Comment:
This seems kinda unnecessary, but I was trying to make it easy for other filter types who may not fit into these three categories perfectly (i.e. they just return in their `_getSearchType` function because search doesn't make sense for them, or they could have multiple search types and just tell the filter to make sure search is "on")